lib/graphql-hive/client.rb

```diff
@@ -20,23 +20,39 @@ def send(path, body, _log_type)
             path: path
           )
 
+        http = setup_http(uri)
+        request = build_request(uri, path, body) 
+        response = http.request(request)
+
+        @options[:logger].debug(response.inspect)
+        @options[:logger].debug(response.body.inspect)
+      rescue StandardError => e
+        @options[:logger].fatal("Failed to send data: #{e}")
+      end
+
+      def setup_http(uri)
         http = ::Net::HTTP.new(uri.host, uri.port)
         http.use_ssl = @options[:port].to_s == '443'
```

Comment:
in the [docs](https://the-guild.dev/graphql/hive/docs/specs/usage-reports), **https** is used. should we ensure that here? only thing is that `#send` is also used with the `/registry` path (not sure if that allows either http and https)
```suggestion
        http.use_ssl = true
```

Reply:
I think this is an option to have it as false because the hive server / database can be self-hosted in many scenarios:
- within a company's AWS environment
- in automated integration tests 
- running hive client and server locally when developers use them

In a lot of these scenarios, SSL won't be used, and just a `http:` protocol.

Reply:
that makes sense! i will keep the option then 🙏